opensafely/pull.py

```diff
@@ -133,6 +157,8 @@ def remove_deprecated_images(local_images):
 def get_default_version_for_image(name):
     if name in ["databuilder", "ehrql"]:
         return "v0"
+    elif name == "python":
+        return "v2"
     else:
         return "latest"
```

Comment:
You could absolutely use a case statement here!  It would improve just about nothing but you _could_ 😜

Reply:
Sadly not, as opensafely-cli runs on user's installed python, so we have to support old python (current lower bound is 3.9, IIRC)

Reply:
booooo (but totally understandable)